interface/forms/group_attendance/save.php

```diff
@@ -0,0 +1,109 @@
+<?php
+include_once("../../globals.php");
+include_once("$srcdir/api.inc");
+include_once("$srcdir/forms.inc");
+include_once("functions.php");
+
+//Get relevant data from group appt (the appt that created the group encounter)
+$appt_data = get_appt_data($encounter);
+
+//Get relevant data from group encounter
+$group_encounter_data = get_group_encounter_data($encounter);
+
+//If saving new form
+if($_GET['mode'] == 'new') {
+
+    //Get the number that should be the new form's id
+    $newid = largest_id_plus_one('form_therapy_groups_attendance');
+
+    //Insert into 'forms' table
+    addForm($encounter, "Group Attendance Form", $newid, "group_attendance", null, $userauthorized);
+
+    //Insert into form_therapy_groups_attendance table
+    $sql_for_table_ftga = "INSERT INTO form_therapy_groups_attendance (id, date, group_id, user, groupname, authorized, encounter_id, activity) " .
+        "VALUES(?,NOW(),?,?,?,?,?,?);";
+    $sqlBindArray = array();
+    array_push($sqlBindArray, $newid, $therapy_group, $_SESSION["authUser"], $_SESSION["authProvider"], $userauthorized, $encounter, '1');
+    sqlInsert($sql_for_table_ftga, $sqlBindArray);
+
+    $patientData = $_POST['patientData'];
+    foreach ($patientData as $pid => $patient){
+
+        //Insert into therapy_groups_participants_attendance table
+        insert_into_tgpa_table($newid, $pid, $patient);
+
+        //Check if to create appt and encounter for each patient (if has certain status and 'bottom' submit was pressed, not 'add_patient' submit).
+        $create_for_patient = if_to_create_for_patient($patient['status']);
+        if($create_for_patient && empty($_POST['submit_new_patient'])){
+
```

Comment:
We aren't sure how to use this function and what it does in the system, as we've never implemented it. Would you be able to elaborate more on the tracker functionality (what it does, where we can see it in action in the system, etc.)?  We will happily embed it in the code.
thanks,
shachar